.github/scripts/versioning.py

```diff
@@ -104,9 +104,9 @@ def extract_version_number_from_file(file_path):
         match = re.search('\s*\*\s*(Amazon FreeRTOS.*V(.*))', content, re.MULTILINE)
         # Is it a kernel file?
         if match is None:
-            match = re.search('\s*\*\s*(FreeRTOS Kernel.*V(.*))', content, re.MULTILINE)
+            match = re.search('\s*\*\s*(FreeRTOS Kernel.*V([0-9]*\.[0-9]*\.[0-9]*))', content, re.MULTILINE)
```

Comment:
Not a blocker, but would be great to paste comments with each expression that provide an example string the regex is looking for. In this case it is fairly obvious, it just tends to make maintenance a bit easier